src/test/spec/mod.rs

```diff
@@ -103,9 +103,14 @@ where
     G: Future<Output = ()>,
     T: DeserializeOwned,
 {
-    let json: Value = serde_json::from_reader(File::open(path.as_path()).unwrap())
+    let mut json: Value = serde_json::from_reader(File::open(path.as_path()).unwrap())
         .unwrap_or_else(|err| panic!("{}: {}", path.display(), err));
 
+    // TODO RUST-36 Remove this when decimal128 support is implemented
+    if path.ends_with("client-side-encryption/legacy/types.json") {
```

Comment:
It's a very ugly hack to have this here, but I couldn't think of a cleaner way - the stripping has to happen before the json is parsed as bson, so test runner predicates are too late.

Reply:
I think this is independent of that - the field that has the issue is `clientOptions.autoEncryptOpts.schemaMap`, which is itself a `bson::Document`; the test file has some fields in that doc that are decimal128 which currently can't be represented in the `bson` crate datastructures.